fluxcd/README.md

```diff
@@ -1,34 +1,104 @@
+
+1. Edit the `fluxcd.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Fluxcd performance data. See the [sample configuration file][4] for all available configuration options.
+
+    This example demonstrates the configuration:
+
+    ```yaml
+    init_config:
+      ...
+    instances:
+      - openmetrics_endpoint: http://<FLUXCD_ADDRESS>:8080
+    ```
+
+2. [Restart the Agent][5] after modifying the configuration.
+
+<!-- xxz tab xxx -->
+<!-- xxx tab "Docker" xxx -->
+
+#### Docker
+
+##### Metric collection
+
+This example demonstrates the configuration as a Docker label inside `docker-compose.yml`. See the [sample configuration file][4] for all available configuration options.
+
+```yaml
+labels:
+  com.datadoghq.ad.checks: '{"fluxcd":{"instances":[{"openmetrics_endpoint":"http://%%host%%:8080"}]}}'
+```
+
+<!-- xxz tab xxx -->
+<!-- xxx tab "Kubernetes" xxx -->
+
+#### Kubernetes
+
+##### Metric collection
 
+This example demonstrates the configuration as Kubernetes annotations on your Fluxcd pods. See the [sample configuration file][4] for all available configuration options.
 
-1. Install the [developer toolkit][10]
- on any machine.
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: '<POD_NAME>'
+  annotations:
+    ad.datadoghq.com/fluxcd.checks: |-
+      {
+        "fluxcd": {
+          "instances": [
+            {
+              "openmetrics_endpoint": "http://%%host%%:8080",
+            }
+          ]
+        }
+      }
+    # (...)
+spec:
+  containers:
+    - name: 'fluxcd'
+# (...)
+```
 
-2. Run `ddev release build fluxcd` to build the package.
+<!-- xxz tab xxx -->
+<!-- xxz tabs xxx -->
 
-3. [Download the Datadog Agent][2].
+Fluxcd metrics are available on the OpenMetrics endpoint. Additionally, Fluxcd allows you to [export custom application-level metrics][10]. You can configure the Fluxcd integration to collect these metrics using the `extra_metrics` option. All Fluxcd metrics, including your custom metrics, use the `fluxcd.` prefix.
 
-4. Upload the build artifact to any host with an Agent and
- run `datadog-agent integration install -w
- path/to/fluxcd/dist/<ARTIFACT_NAME>.whl`.
+**Note:** Custom Fluxcd metrics are considered standard metrics in Datadog.
 
-### Configuration
+This example demonstrates a configuration leveraging the `extra_metrics` option:
 
-1. Edit the `fluxcd.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your fluxcd performance data. See the [sample fluxcd.d/conf.yaml][4] for all available configuration options.
+```yaml
+init_config:
+  ...
+instances:
+  - openmetrics_endpoint: http://<FLUXCD_ADDRESS>:8080
+    # Also collect your own Fluxcd metrics
+    extra_metrics:
+      - my_custom_fluxcd_metric
+```
 
-2. [Restart the Agent][5].
+More info on how to configure this option can be found in the [sample `fluxcd.d/conf.yaml` configuration file][11].
```

Comment:
Can you verify if this should point to the [4] link? There's no hyperlink for [11].
```suggestion
See the [sample configuration file][4] for all available configuration options
```